src/main/java/appeng/client/gui/implementations/GuiCraftConfirm.java

```diff
@@ -111,16 +127,32 @@ boolean isAutoStart()
 		return ( (ContainerCraftConfirm) this.inventorySlots ).isAutoStart();
 	}
 
+	private void reinitalize()
+	{
+		this.buttonList.clear();
+		this.initGui();
+	}
+
 	@Override
 	public void initGui()
 	{
+		final int staticSpace = ITEMSTACK_TOP_OFFSET + CPU_TOP_OFFSET;
+
+		calculateRows(staticSpace);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) '(' is not followed by whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) ')' is not preceded with whitespace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)